generator/sbpg/targets/resources/sbp_base.tex

```diff
@@ -275,6 +276,45 @@ \section{GNSS Signals}
 \captionof{table}{GNSS Signals Table}
 \end{center}
 
+\newpage
+
+\section{Signal and Tracking Mode Identifier}
+\label{sec:swift_signals}
+\begin{large}
+Code, Constellation, and Band. Swift signal descriptions are provided in both RINEX and RTCM Conventions.
+\end{large}\\
+
+\begin{center}
+  \begin{longtable}{l|ll|ll|ll}
+    \toprule
+    \multirow{2}{*}{Signal}& \multicolumn{2}{l|}{GPS}          & \multicolumn{2}{l|}{GALILEO} & \multicolumn{2}{l}{BEIDOU} \\ \cline{2-7}
+                           & Rinex Desc. & RTCM Desc.         & Rinex Desc.  & RTCM Desc.   & Rinex Desc.  & RTCM Desc.  \\
+    \midrule
+    0                      & GPS L1CA    & L1 C/A             & GAL E1A      & E1 A         & BDS2 B1I     & B1 I        \\
+    1                      & GPS L1P     & L1 P               & GAL E1B      & E1 B I/NAV   &              &             \\
+    2                      & GPS L1W     & L1 Z-tracking      & GAL E1C      & E1 C         &              &             \\
+    3                      &             &                    & GAL E1X      & E1 B+C       & BDS3 B3I     & B3          \\
+    4                      &             &                    & GAL E1Z      & E1 A+B+C     & BDS3 B3Q     & B3 Q        \\
+    5                      & GPS L2C     & L2 C/A             & GAL E5I      & E5a I F/NAV  & BDS3 B3X     & B3 I+Q      \\
+    6                      & GPS L2D     & L2 L1(C/A)+(P2-P1) & GAL E5Q      & E5a Q        & BDS2 B2      & B2 I        \\
+    7                      & GPS L2CM    & L2 L2C (M)         & GAL E5X      & E5a I+Q      &              &             \\
+    8                      & GPS L2CL    & L2 L2C (L)         & GAL E7I      & E5b I I/NAV  &              &             \\
+    9                      & GPS L2CX    & L2 L2C (M+L)       & GAL E7Q      & E5b Q        &              &             \\
+    10                     & GPS L2P     & L2 P               & GAL E7X      & E5b I+Q      &              &             \\
+    11                     & GPS L2W     & L2 Z-tracking      & GAL E8I      & E5 I         &              &             \\
```

Comment:
I omitted BDS3 signals (7 to 11) since they have a defined Rinex and RTCM code, they are not supported yet. Is this ok?